### PR TITLE
Add CVE-2021-28689 XSA-370

### PR DESCRIPTION
**Pull request by xen-project-security-team**
Signed-off-by: Ian Jackson <iwj@xenproject.org>

### CODE REVIEW

File: 2021/26xxx/CVE-2021-28689.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (cve-team) [inline 2021/26xxx/CVE-2021-28689.json]: Expected location (2021/28xxx/CVE-2021-28689.json) of file does not match actual location (2021/26xxx/CVE-2021-28689.json)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/26xxx/CVE-2021-28689.json (hunk 1 above)
Comment (xen-project-security-team) [reply 2021/26xxx/CVE-2021-28689.json]: fixed

**Comment by xen-project-security-team** [comment]
ping?

**Comment by xen-project-security-team** [comment]
Please re-review this?

File: 2021/28xxx/CVE-2021-28689.json [hunk 2]
```diff
@@ -1,18 +1,116 @@
 {
```
Comment (cve-team) [inline 2021/28xxx/CVE-2021-28689.json:1]: None is not of type 'string'

Comment (cve-team) [inline 2021/28xxx/CVE-2021-28689.json:1]: None is not of type 'string'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by xen-project-security-team** [comment]
Fixed, again, sorry.

**Pull request merged**